front/logviewer.php

```diff
@@ -50,16 +50,20 @@
 if (($_GET['action'] ?? '') === 'download') {
     $logparser = new LogParser();
     $logparser->download($filepath);
-} elseif (($_POST['action'] ?? '') === 'empty') {
+} elseif (($_REQUEST['action'] ?? '') === 'empty') {
     $logparser = new LogParser();
     $logparser->empty($filepath);
     Html::back();
+} elseif (($_REQUEST['action'] ?? '') === 'delete') {
```

Comment:
This is a writing operation, CSRF token have to be validated
```suggestion
} elseif (($_POST['action'] ?? '') === 'delete') {
```